app/scripts/background.js

```diff
@@ -193,6 +196,15 @@ const {
   reject: rejectInitialization,
 } = deferredPromise();
 
+// Define this function at the top level, before it's used
```

Comment:
Nit: conventionally we include a block comment above each function to explain what it does, and what the inputs/outputs are.

This comment seems to be calling out that we're defining the function earlier, but, it's unclear why we're doing this, or why it would be noteworthy. Perhaps we should replace this with a more standard JSDoc block instead.

Reply:
Thanks for reviewing ~ it is addressed 9e038b3112f6bc6cfbcf616c406488b14b869ff2 to make it more clear